package.json

```diff
@@ -39,15 +39,19 @@
     "clean:docs": "rimraf **/_api-extractor-temp docs/api/*/**",
     "clean:nyc": "rimraf nyc/**",
     "clean:r11s": "cd server/routerlicious && npm run clean",
-    "format": "lerna run format --parallel --stream",
+    "format": "npm run prettier:fix:root && lerna run format --parallel --stream",
     "preinstall": "npx only-allow pnpm",
     "layer-check": "fluid-layer-check --info build-tools/packages/build-tools/data/layerInfo.json",
     "lerna": "lerna",
-    "lint": "lerna run lint --no-sort --stream",
-    "lint:fix": "lerna run lint:fix --no-sort --stream",
+    "lint": "npm run prettier:root && lerna run lint --no-sort --stream",
+    "lint:fix": "npm run prettier:root:fix && lerna run lint:fix --no-sort --stream",
     "policy-check": "flub check policy --exclusions build-tools/packages/build-tools/data/exclusions.json",
     "policy-check:asserts": "flub check policy --exclusions build-tools/packages/build-tools/data/exclusions.json --handler assert-short-codes --fix",
     "policy-check:fix": "flub check policy --exclusions build-tools/packages/build-tools/data/exclusions.json --excludeHandler assert-short-codes --fix",
+    "prettier": "npm run prettier:root && lerna run prettier --no-sort --stream",
```

Comment:
We probably need to add a prettier config to the repo root if we're adding root linting.

Reply:
Yeah, good catch. CI failed, but I can't figure out why it was working locally. \<shrug\>